core/src/main/java/org/apache/accumulo/core/security/ColumnVisibility.java

```diff
@@ -502,7 +520,36 @@ public ColumnVisibility(Text expression) {
    * @see #ColumnVisibility(String)
    */
   public ColumnVisibility(byte[] expression) {
-    validate(expression);
+    this(expression, true);
+  }
+
+  /**
+   * Creates a column visibility for a mutation from a string already encoded in UTF-8 bytes. Allows
+   * validation to be skipped for use with accumulo-access AccessExpressions
+   *
+   * @param expression visibility expression, encoded as UTF-8 bytes
+   * @param validate enables or disables validation and parse tree generation
+   */
+  private ColumnVisibility(byte[] expression, boolean validate) {
```

Comment:
need to change the `validate` variable name to avoid conflicting with the `validate` method